### DISCUSSION TITLE
Setting up OpenSnitch to work on a remote server

### DISCUSSION THREAD

**Discussion by Doroshenko-T** [evilsocket/opensnitch#1084] [Q&A]
Hi everybody,

How do I set up OpenSnitch so that its daemon is located on a remote server but its client is on the desktop?
This is very necessary when working via remote Socks5.

**Comment by gustavo-iniguez-goya** [comment]
hey @T-Shilov ,

Check the docs: https://github.com/evilsocket/opensnitch/wiki/Nodes

**Comment by gustavo-iniguez-goya** [comment]
By the way, I've been working on migrating the cli tool https://github.com/gustavo-iniguez-goya/opensnitch/issues/54 to v1.6.x, I hope to push it soon:

[opcli-black-theme.webm](https://github.com/evilsocket/opensnitch/assets/2742953/b8d67ac4-5175-4065-8d20-c7aa735b0071)
